todo_app/app_config.py

```diff
@@ -0,0 +1,10 @@
+import os
```

Comment:
This file is obsolete now that we're using Trello to store todos

Reply:
File removed